cmd/codegraph/git.go

```diff
@@ -5,22 +5,23 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/cayleygraph/cayley/quad/nquads"
 	"io"
 	"os"
 	"strings"
 
-	"github.com/MLonCode/codegraph/git"
+	"github.com/cayleygraph/cayley/quad/nquads"
+
+	"github.com/mloncode/codegraph/git"
 	"github.com/spf13/cobra"
 )
 
 func init() {
-	var g *git.GitGraph
+	var g *git.Graph
 	cmdGit := &cobra.Command{
 		Use:   "git <command>",
 		Short: "Git-related commands",
 	}
-	db := cmdGit.PersistentFlags().StringP("db", "a", "./", "database directory")
+	db := cmdGit.PersistentFlags().StringP("db", "d", "./", "database directory")
```

Comment:
Can we leave it as `-a`? The reason is that if we continue to use Cayley, it uses `-d` for the database type and `-a` for the database path (**a**ddress for the remote databases).

Reply:
ok